src/Codeception/Module/Yii2.php

```diff
@@ -628,18 +633,21 @@ public function grabFixture(string $name, null|string $index = null): Fixture|\y
      * $user_id = $I->haveRecord('app\models\User', array('name' => 'Davert'));
      * ?>
      * ```
+     *
      * @template T of \yii\db\ActiveRecord
-     * @param class-string<T> $model
-     * @param array<string, mixed> $attributes
-     * @part orm
+     * @param    class-string<T>      $model
+     * @param    array<string, mixed> $attributes
+     * @part     orm
```

Comment:
```php
* @return int|string|array Primary key of the created record
```

Reply:
among other things, please replace `array()` with `[ ]` in the docblocks,
php closing tags are also not necessary.

Reply:
What do you mean with the array thing? You dont mean array<x,y> right?